niworkflows/interfaces/filter.py

```diff
@@ -0,0 +1,1165 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should end with a period, question mark, or exclamation point (not 's')](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096913)